core/lib/fields/richText.ts

```diff
@@ -8,3 +14,98 @@ export const serializeProseMirrorDoc = (value: Node) => {
 	// We stringify and then parse in order to make it a plain JSON object
 	return JSON.parse(JSON.stringify(value.toJSON()));
 };
+
+interface GetPubValuesReturnType {
```

Comment:
something like this should go in the editor's `getPubValues` func instead—I put it here mostly to help myself write the parse function below